src/manifest.ts

```diff
@@ -463,26 +476,123 @@ export class Manifest {
     return pr;
   }
 
-  async githubRelease(): Promise<GitHubReleaseResponse[] | undefined> {
+  async githubRelease(): Promise<
```

Comment:
Am I reading this correctly that, unlike the current approach where we look specifically for merged pull requests that have `autorelease: pending`, `githubRelease()` simply pulls the most recently merged pull request, uses this manifest as a starting point, and performs the release logic?

What would trigger `githubRelease()`?

Reply:
`githubRelease` will be called by `manifest-release` cli cmd.

Good catch, I should check the label! The way it's currently setup yes: it will only get the very latest merged "release PR"* since `lastMergedPRByHeadBranch` returns the most recent PR merged to `defaultBranch` from [`getBranchName()`](https://github.com/googleapis/release-please/blob/c24d0414edde8d6011d4b046d189002d36739277/src/manifest.ts#L94) regardless of tags. So multiple repeated runs will just keep getting the last merged PR and trying to re-create the tags/releases. I'll definitely add checking for the `auto-release:pending` to prevent that in this PR.

*I suppose there's a small hole here: if, prior to release-please creating the PR branch,  someone were to manually create a branch with exactly the same name as [`getBranchName()`](https://github.com/googleapis/release-please/blob/c24d0414edde8d6011d4b046d189002d36739277/src/manifest.ts#L94) (e.g. `release-please/branches/main`) and merge some random PR from it before release-please comes and clobbers their stuff by force-pushing the branch, then we'd have a non-release-please created PR turning up in our query. I think the solution is the same as above: just add checking for `auto-release:pending` will safeguard against this edge case. If someone manually selects our branch name for their random branch AND applies the `auto-release:pending` tag, then that's on them!

This "accidental" branch name shadowing case as well as the case you mentioned before of "forgetting to do a `manifest-release` run between release PR merges" leads me to think we need to consider iterating back through merged release PRs till we find one that is `auto-release:tagged` and then roll forward creating release/tags per release PR we found since the last `auto-release:tagged` PR that have `auto-release:pending`. Would you be ok with adding this robustness in a later PR?

Reply:
> Would you be ok with adding this robustness in a later PR?

For sure, I think in practice we'll find going to the last release PR is sufficient, since we'll eventually have the behavior wired up to GitHub actions that fire within a few moments of a merge.

I think we're on the same page that we were just missing the label check 😄